outputs.tf

```diff
@@ -223,3 +223,8 @@ output "username" {
   sensitive   = true
   value       = azurerm_kubernetes_cluster.main.kube_config[0].username
 }
+
+output "web_app_routing_identity" {
+  description = "The `azurerm_kubernetes_cluster`'s `web_app_routing_identity` block."
```

Comment:
I'd like to address that this block is a list of object:

description = "The `azurerm_kubernetes_cluster`'s `web_app_routing_identity` block, it's type is a list of object."

Reply:
Done